aria/.pylintrc

```diff
@@ -77,7 +77,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
```

Comment:
meta comments:
1) create JIRA issue for supporting custom workflow logs i.e. during execution time (but not graph building)
2) decide whether to support passing custom handlers to workflow as well, right now only operation is supported - possibly add jira issue on this as well, allowing for workflow logger to pass onto operations